buildpacks/php/src/main.rs

```diff
@@ -0,0 +1,131 @@
+#![warn(clippy::pedantic)]
```

Comment:
Running `cargo clippy --all-targets` on this branch locally with Rust 1.70, I get 62 warnings across the two crates, of which 38 are auto-correctable using the `--fix` mode. Just checking that:
- you know about `--fix` (in case it saves you some time)
- you know about the whole IDE `check` vs `clippy` command config thing (IDEs tend to default to running the `check` command rather than the `clippy` command, but for VSCode at least can be configured to run the latter, if avoiding surprises in CI is preferred)